### DISCUSSION TITLE
Dairies Emission Factor inclusion

### DISCUSSION THREAD

**Discussion by arby1005** [USEPA/supply-chain-factors#29] [Q&A] [answered]
Hi all,

I am calculating the food emissions which include milk, yoghurt and lactic drink, etc. However, I found that the fluid milk emission factor and dairies emission factor vary quite a lot (1.xx kgCO2e/USD vs 3.xx kgCO2e/USD). For yoghurt and lactic drink, due to no specific emission factor to be applied, dairies emission factor was used. However, I doubt if it should be the way to do so. 

1) What dairies product under dairies category contribute to a large emission factor?
2) For lactic drink and yoghurt, which emission factor do you suggest to use?

Thank you.

**Comment by MoLi7** [comment, marked as answer]
Hi @arby1005 

I understand that some BEA sector names are too brief to disclose detailed products. Good news is, the classification of dairy products in BEA and NAICS systems are aligned quite well, so it is recommended to use NAICS definition to help you confirm which BEA sector yogurt and lactic drink belong to.

In this [hierarchical NAICS documentation](https://www.census.gov/naics/?input=31151&year=2012&details=31151), you will find yogurt and flavored milk drinks under 311511 which corresponds to BEA 31151A - Fluid milk and butter. Therefore, it is suggested that you should use the emission factors of 31151A for yogurt and lactic drinks.

Hope this helps!
